redirects.json

```diff
@@ -121,7 +121,7 @@
   },
   {
     "source":"/ui/storage/s3-image-picker/q/framework/<platform>/",
-    "target":"https://ui.docs.amplify.aws/components/storage?platform=<platform>#s3-image-picker",
+    "target":"https://ui.docs.amplify.aws/react/connected-components/storage/storagemanager",
```

Comment:
https://ui.docs.amplify.aws/components/storage?platform=<platform>#s3-image-picker 404s; I think this is the closest link